src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsEventSourceLifetime.cs

```diff
@@ -21,6 +21,7 @@ public class TestsEventSourceLifetime
         /// </summary>
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)] // non-Windows EventSources don't have lifetime
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "ExerciseEventSource use private member reflection")]
```

Comment:
You would either need to build with a static reference (using InternalsVisibleTo) or make the private types/members public for now in order to make them reflectable. We're planning to have other ways to make them reflectable, but that'll take some time to be ready.